tests/test_cli_compile.py

```diff
@@ -1534,3 +1534,69 @@ def test_duplicate_reqs_combined(
     assert out.exit_code == 0, out
     assert str(test_package_2) in out.stderr
     assert "test-package-1==0.1" in out.stderr
+
+
+@pytest.mark.parametrize(
+    ("pkg2_install_requires", "req_in_content", "out_expected_content"),
+    (
+        pytest.param(
+            "",
+            ["test-package-1===0.1.0\n"],
+            ["test-package-1===0.1.0"],
+        ),
```

Comment:
```suggestion
        pytest.param(
            "",
            ["test-package-1===0.1.0\n"],
            ["test-package-1===0.1.0"],
            id="{what do we test here}"
        ),
```

It would be useful to have a descriptive ID on each test.

Reply:
Done